assets/css/main.css

```diff
@@ -576,7 +576,8 @@ a {
       background-position: center left; }
     .dataTables_wrapper table.dataTable.no-footer {
       border: none; }
-    .dataTables_wrapper table.dataTable.domain td:first-child {
+    .dataTables_wrapper table.dataTable.domain td.odd:first-child,
+    .dataTables_wrapper table.dataTable.domain td.even:first-child {
```

Comment:
It looks like the CSS changes here might have been a result of how I addressed the merge conflict earlier in #169. The changes here are correct and should be merged in with this PR.